packages/cache-manager/index.ts

```diff
@@ -5,16 +5,16 @@ export type IDataProviderCallback = () => string | Promise<string>;
 
 export interface CacheManager {
     getItem: (key: string) => Promise<string>;
-    setItem: (key: string, value: string) => Promise<void>;
+    setItem: (key: string, value: string, ttl:number) => Promise<void>;
     wrap: (key: string, fallback: IDataProviderCallback) => Promise<string>;
 }
 
 export function createCacheManager(cacher: ICache, { ttl }: CacheManagerOptions = { ttl: 600 }): CacheManager {
 
     return {
         getItem: cacher.getItem,
-        setItem(key: string, value: string) {
-            return cacher.setItem(key, value, { ttl });
+        setItem(key: string, value: string, itemTtl :number =ttl) {
```

Comment:
you shoudl also get the entire config object at a third variable and put the defaults from the above closure.

Reply:
whats the name of this syntax - { ttl }: CacheManagerOptions ? I mean - without set object name 
how can I get it from the above closure.

Reply:
options: CacheManagerOptions = {ttl}